### PR TITLE
DECO2500

### PR DESCRIPTION
**Pull request by chrisw310**
Review for DECO2500

### CODE REVIEW

**Comment by wisebaldone** [comment]
LGTM

Content is good, formatting uses indented paragraphs with also a line break, which is a little odd but easily fixed later.

**Comment by draganmarjanovic** [comment]
LGTM

File: tex/courses/subjects/DECO2500.tex [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
+
+    Assessment involves weekly practicals, a research paper, and a final exam. 40\% of the grade are weekly pracs worth 4\% each. These weekly pracs can be easy marks if you prepare beforehand. As with any group work, be sure to find good people to work with in the first week. \\
+
+    The second big assessment is the research document from a choice of 20 topics. Some involve real-world testing, and some are just literature reviews. Feel free to pick whatever you'd like, but some topics are easier than others. \\
+
+    The final exam will quiz you on the textbook. Make sure to read it cover to cover, as you will be quizzed on specific case studies in the book. Past exam questions will be your friend when studying for the final exam. \\
+
+    Penelope Sanderson is a lecturer from the Psychology department. If you approach the course as more of a psychology course than an engineering course, then you will have better expectation and understanding of the activities. Not everyone will enjoy this course, but the course gives a solid foundation to understanding how usabiltiy testing makes for better software.
```
Comment (CameronAavik) [inline tex/courses/subjects/DECO2500.tex:23]: TYPO: usability*

**Comment by TRManderson** [comment]
I feel like we should mention how terrible the course used to be

**Comment by wisebaldone** [comment]
2015 was a once off, I believe penny use to run it before then so this would be the typical

**Pull request merged**